### ISSUE TITLE
New analyzer | warning about duplicate fields in fieldgroup

### ISSUE THREAD

**Issue by ernestasjuska** [StefanMaron/BusinessCentral.LinterCop#1189]
Hi,

Today I learned that you can specify same field multiple times in the same table group and the compiler does not complain. When you use dropdown in web client, it simply crashes. Interestingly, it only happens when table relation is specified on a page field (report request page field in our case) - table field does not crash web client. I could reproduce this in BC26 both on prem and BC online.

Please add a code analyzer for this case.

Code example - duplicate Starting Date field:
```al
table 70302081 "LBC VAT Period"
{
    // ...
    fieldgroups
    {
        fieldgroup(DropDown; "Starting Date", Name, "New VAT Year", Closed, "Starting Date", "Date Locked")
        {
        }
        fieldgroup(Brick; "Starting Date", Name, "New VAT Year", Closed, "Starting Date", "Date Locked")
        {
        }
    }
// ...
```

Thanks.

**Comment by pri-kise** [comment]
I think that this should be reported in the AL Language repository: https://github.com/Microsoft/AL

The best would be to provide a MWE where the error can be reproduced.

Additionally: Please do create a discussion for new rule suggestions:
(See also: https://github.com/StefanMaron/BusinessCentral.LinterCop?tab=readme-ov-file#please-contribute )
